src/robusta/core/sinks/robusta/dal/supabase_dal.py

```diff
@@ -277,6 +279,80 @@ def get_active_services(self) -> List[ServiceInfo]:
             for service in res.get("data")
         ]
 
+    def get_active_nodes(self) -> List[NodeInfo]:
+        res = (
+            self.client.table(NODES_TABLE)
+            .select(
+                "name", "node_creation_time", "taints", "conditions", "memory_capacity", "allocatable_memory",
+                "allocated_memory", "cpu_capacity", "allocatable_cpu", "allocated_cpu", "pods_count", "pods",
+                "internal_ip", "external_ip", "node_info",
+            )
+            .filter("account_id", "eq", self.account_id)
+            .filter("cluster_id", "eq", self.cluster)
+            .filter("deleted", "eq", False)
+            .execute()
+        )
+        if res.get("status_code") not in [200]:
+            msg = f"Failed to get existing nodes (supabase) error: {res.get('data')}"
+            logging.error(msg)
+            self.handle_supabase_error()
```

Comment:
Can we upgrade to a newer supabase-py version? I think the refresh error has been fixed and we can stop using `self.handle_supabase_error()` in newer versions: https://github.com/supabase-community/gotrue-py/issues/9

Reply:
Opening a separate issue for that, it's unrelated to this specific task

Reply:
Yep, thanks